<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="InsertButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
     </font>
    </property>
    <property name="text">
     <string>insert</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>введите элемент для добавления</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindMaxStackButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FindMaxStack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="indexLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>введите индекс для элемента</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sizeLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ввод размера генерации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>31</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ввода ключа для поиска</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="removeLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>650</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ввода ключа для удаления</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
